src/agent/src/image.rs

```diff
@@ -63,6 +63,27 @@ impl ImageService {
             image_client.config.auth = true;
         }
 
+        #[cfg(feature = "guest-pull")]
+        {
+            let enable_signature_verification = &AGENT_CONFIG.enable_signature_verification;
+            debug!(sl(), "Enable image signature verification: {:?}", enable_signature_verification);
+            image_client.config.security_validate = enable_signature_verification.clone();
+        }
+
+        #[cfg(feature = "guest-pull")]
+        if !&AGENT_CONFIG.image_policy_file.is_empty() {
+            let image_policy_file = &AGENT_CONFIG.image_policy_file;
+            debug!(sl(), "Use imagepolicy file {:?}", image_policy_file);
+            image_client.config.file_paths.policy_path = image_policy_file.clone();
+        }
+
+        #[cfg(feature = "guest-pull")]
+        if !&AGENT_CONFIG.simple_signing_sigstore_config.is_empty() {
+            let simple_signing_sigstore_config = &AGENT_CONFIG.simple_signing_sigstore_config;
+            debug!(sl(), "Using RedHat Simple Signing image signature file {:?}", simple_signing_sigstore_config);
```

Comment:
Hi @Xynnn007 !

I'm trying to figure out if saying "RedHat Simple Signing" is the most adequate name. Sorry in advantage, this is a topic outside of my scope of expertise...

I don't know the full history (likely @stevenhorsman knows) so I was digging into image-rs docs. I found a reference to https://www.redhat.com/en/blog/container-image-signing who author said the simple signing mechanism would be contributed to [OCI's image library](https://github.com/containers/image) eventually. So not sure if we can claim it OCI compliance instead...

@bpradipt is this signature mechanism used in Openshift? How's it called?

Reply:
Oh, yes. Because some documents also say it with "RedHat" prefix, so I added a prefix here. In order to avoid ambiguity, let me delete the RedHat prefix - I believe everyone can understand what SimpleSigning refers to.

Reply:
Should simple signing be part of this change as I don't think we have the test cases and have another issue to cover this?

Reply:
Saying that it's not really doing much harm, so I'm not against it being it as long as we don't document that we support simple signing without testing. I think we can also do some improvements on the image-rs side for it as well after some support got merged into the `oci-distribution` crate

Reply:
I deleted simple signing related things in the code. Thus resolve this conversation.